src/trio/_tests/test_exports.py

```diff
@@ -157,6 +157,9 @@ def no_underscores(symbols: Iterable[str]) -> set[str]:
         ast = linter.get_ast(module.__file__, modname)
         static_names = no_underscores(ast)  # type: ignore[arg-type]
     elif tool == "jedi":
+        if sys.implementation.name != "cpython":
+            pytest.skip("jedi does not support pypy")
```

Comment:
```suggestion
        if sys.implementation.name != "cpython":  # pragma: no branch
            pytest.skip("jedi does not support pypy")
```

Reply:
This feels weird though, like I don't think codecov should be complaining about this :(